ledis/Views/CustomTextField.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CustomTextField.swift
+//  ledis
+//
+//  Created by Maya Vu on 30.06.18.
+//  Copyright © 2018 Maya Vu. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
You can learn some info about custom controls from this video:
[drawing in iOS](https://videos.raywenderlich.com/courses/133-drawing-in-ios/lessons/5)